apps/meteor/server/services/team/service.ts

```diff
@@ -77,6 +78,28 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 		};
 
 		try {
+			let roomId = room.id;
+			if (roomId) {
+				await Message.saveSystemMessage('user-converted-to-team', roomId, team.name, createdBy);
```

Comment:
We shouldn't send a message before doing the actions cause if some of the actions fail or we error out later, the message will still be there.

Reply:
Nice catch!